example/package.json

```diff
@@ -5,11 +5,11 @@
   "private": true,
   "main": "./dist/index.js",
   "dependencies": {
-    "@sparkdesignsystem/spark-core": "8.0.0",
-    "@sparkdesignsystem/spark-extras": "3.1.7",
-    "@sparkdesignsystem/spark-card": "4.0.0",
-    "@sparkdesignsystem/spark-highlight-board": "4.0.0",
-    "@sparkdesignsystem/spark-description-table": "3.1.7",    
+    "@sparkdesignsystem/spark-core": "9.0.0",
```

Comment:
Not really important, but I'm not sure why these are deps? Since the example app just pulls everything in from `react-spark`, which has these deps

Reply:
i don't fully understand it either, but it's the only thing that seems to get all the apps working correctly